src/main/java/com/google/sps/servlets/MealServlet.java

```diff
@@ -57,16 +94,17 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         // Get meal
         // GET meal/
         if (pathInfo == null || pathInfo.equals("/")) {
-            searchMeal(request, response);
+            getMealList(request, response);
```

Comment:
I don't think the search and the meal listing should be separated.

Both functionalities will list meals for the user. The difference is that search will return only some of the meals, while a regular GET returns all of them.